webui/react/src/pages/F_ExpList/glide-table/MultiSortMenu.tsx

```diff
@@ -183,7 +183,7 @@ const MultiSortRow: React.FC<MultiSortRowProps> = ({ sort, columns, onChange, on
       </div>
       <div>
         <Button
-          icon={<Icon name="close" />}
+          icon={<Icon name="close" title="close" />}
```

Comment:
**nit:** From what I've seen in the previous Icon PR, the titles were sentence casing.

Reply:
Yeah, they were. It would also be good to make this a little more clear, something like `Remove sort`